### ISSUE TITLE
[1866] Buying a train from a friend after EMR is inconsistent

### ISSUE THREAD

**Issue by physic** [tobymao/18xx#7871]
Case 1: Take a loan to EMR a train. Afterward, buying a train from another company is prohibited.
Case 2: Issue a share to EMR a train. Afterward, buying a train from another company is allowed.

Based on my reading, neither of these cases are prohibited (or in fact addressed) by the rules. Both cases should be allowed.

https://18xx.games/game/86210?action=484

**Comment by philcampeau** [comment]
Rule 12.8.1 states: 

> Cannot use EMR to pay for a train purchased from another company. 

So if anything, neither issuing nor loans should be allowed if cross buying.

**Comment by physic** [comment]
> Rule 12.8.1 states:
> 
> > Cannot use EMR to pay for a train purchased from another company.
> 
> So if anything, neither issuing nor loans should be allowed if cross buying.

Yes. However, in my mind there is a distinction between 1) EMR to cross buy from another corp, and 2) cross buy from another corp after EMR, by using remaining funds.

So we potentially have seven cases for deciding whether buying a train from another corp should be allowed.

EMR (no loans, no shares issued, i.e. yes money from President): Currently allowed. Should not be allowed. ❌
* https://gist.github.com/physic/4515bbc7d0002c8d2b0170e4b7f72fc1 Action 701.
EMR (yes loans, no shares issued): Currently not allowed. Should not be allowed. ✔
EMR (no loans, yes shares issued): Currently allowed. Should not be allowed. ❌
EMR (yes loans, yes shares issued): Currently not allowed. Should not be allowed. ✔
Buy after EMR (no loans, no shares issued): N/A. This case is not possible, because the corp would have £0.
Buy after EMR (yes loans, no shares issued): Currently not allowed. Should maybe be allowed. ❌❔
Buy after EMR (no loans, yes shares issued): Currently allowed. Should maybe be allowed. ✔❔
Buy after EMR (yes loans, yes shares issued): Currently not allowed. Should maybe be allowed. ❌❔

**Comment by Lime-Turtle** [comment]
If a company has to EMR to fund a train, then the train purchased must be from the bank (or bank pool).
Having done that, if the company then has any money remaining in treasury, it may buy a train from another company using that remaining money.

**Comment by philcampeau** [comment]
> I've asked for a clarification as a comment to the rules document. (I would have guessed that neither case should be allowed, but we'll see.)

@dfannius did you ever get a reply? Without further clarification, we should close this report.

**Comment by philcampeau** [comment]
No update from designer. Closing until we have confirmation of the rule.
